app/views/ExposureHistory/DetailedHistory.js

```diff
@@ -13,6 +14,11 @@ import { SingleExposureDetail } from './SingleExposureDetail';
  */
 export const DetailedHistory = ({ history }) => {
   const exposedDays = history.filter(day => day.exposureMinutes > 0);
+  const wtdmParaText =
```

Comment:
what do you think about not using an abbreviation here? something like: ` const whyThisMattersText =`

Reply:
I think that is a good idea! Will do